### DISCUSSION TITLE
How do we host the custom Frappe Insights app in azure? Any suggestions?

### DISCUSSION THREAD

**Discussion by Phanidaptify** [frappe/insights#353] [General]
I am trying to create customizations to frappe insights. Want to deploy into azure. Any leads will be helpful

**Comment by nextchamp-saqib** [comment]
Docker setup is the easiest way to get started. Check out the readme for [installation steps](https://github.com/frappe/insights?tab=readme-ov-file#docker-recommended).

**Reply by Phanidaptify** [reply]
Not the local setup. I am able to setup my local using docker. I want to host an application with few customization into another domain. Any leads on that?
